gdal.cmake

```diff
@@ -353,7 +353,7 @@ if (GDAL_ENABLE_MACOSX_FRAMEWORK)
 else ()
   include(GNUInstallDirs)
   set(INSTALL_PLUGIN_DIR
-      "lib/gdalplugins"
+      "lib${LIB_SUFFIX}/gdalplugins"
```

Comment:
With `GNUInstallDirs` ahead, shouldn't this be:
~~~cmake
   "${CMAKE_INSTALL_LIBDIR}/gdal_plugins"
~~~

Reply:
@manisandro opinion w.r.t above suggestion of using ${CMAKE_INSTALL_LIBDIR} ?

Reply:
Works for me!

Reply:
PR amended